server/static/index.html

```diff
@@ -1,45 +1,25 @@
-{{define "description"}}
-<h2>Prevent short link services from tracking you</h2>
-<div id="features">
-    <div><img src="/static/img/cookie.svg"><br>Remove cookies</div>
-    <div><img src="/static/img/search.svg"><br>Remove tracking parameters</div>
-    <div><img src="/static/img/warning.svg"><br>Check blacklist</div>
-</div>
-{{end}}
 {{define "action"}}
-<h3>Directly redirect</h3>
-<div class="form">
-    <input type="url" id="urlDirect"
-        placeholder="https://tinyurl.com/yyy63fz9"> <button id="unshortDirectButton">Unshort &amp;
-        Redirect</button>
-</div>
-<h3>Show the redirect url first</h3>
-<div class="form">
-        <input type="url" id="urlShow"
-        placeholder="https://tinyurl.com/yyy63fz9"> <button id="unshortShowButton">Unshort &amp; Show</button>
+<div class="input-group input-group-lg">
+    <input type="text" class="form-control" id="urlDirect" placeholder="https://tinyurl.com/yyy63fz9">
+    <div class="input-group-btn">
+        <button class="btn btn-default" type="submit" id="unshortDirectButton">
+            unshorten
+        </button>
+    </div>
 </div>
 <script>
     let buttonDirectElement = document.getElementById("unshortDirectButton");
     let urlDirectElement = document.getElementById("urlDirect");
     buttonDirectElement.addEventListener("click", function () {
         if (urlDirectElement.value != "") {
             window.location.href = "https:\/\/unshort.link/" + urlDirectElement.value;
-            return
+            return;
         }
-        window.location.href = "https:\/\/unshort.link/https://tinyurl.com/yyy63fz9";
-    });
-    let buttonShowElement = document.getElementById("unshortShowButton");
-    let urlShowElement = document.getElementById("urlShow");
-    buttonShowElement.addEventListener("click", function () {
-        if (urlShowElement.value != "") {
-            window.location.href = "https:\/\/unshort.link/d/" + urlShowElement.value;
-            return
+        if(urlDirectElement.placeholder != "") {
+            window.location.href = "https:\/\/unshort.link/https://tinyurl.com/" + urlDirectElement.placeholder;
```

Comment:
I think `https:\/\/unshort.link/https://tinyurl.com/" + urlDirectElement.placeholder;` should be without the double tinyurl?
`https:\/\/unshort.link/" + urlDirectElement.placeholder;`

Reply:
Whoops. You are correct. Typo on my part.